internal/cli/help.go

```diff
@@ -128,7 +165,8 @@ func (c *helpCommand) HelpTemplate() string {
 
 var (
 	reHelpHeader = regexp.MustCompile(`^[a-zA-Z0-9_-].*:$`)
-	reCommand    = regexp.MustCompile(`"waypoint \w+"`)
+	reCommand    = regexp.MustCompile(`"waypoint (\w\s?)+"`)
+	reFlag       = regexp.MustCompile(`(\s|^)(-[\w-]+)(\s|$)`)
```

Comment:
I needed the below to get the `"-accept-tos"` bit in `install -h` to highlight.
```suggestion
	reFlag       = regexp.MustCompile(`(\s|^|")(-[\w-]+)(\s|$|")`)
```

Reply:
We could instead remove all `"` around flags in help texts.

Reply:
Or just `/W` and assume we won't ever have something like `$-this-is-not-a-flag#`

Reply:
Ah yes. No this is fine with me!